src/content/reference/cli.mdx

```diff
@@ -213,13 +214,16 @@ $ okteto deploy
 | _--timeout_                                 | (duration) | The duration to wait for the development environment deployment to complete. Any value should contain a corresponding time unit e.g. 1s, 2m, 3h (default 5m0s) |
 | _--var_                                     |   (list)   | Set a pipeline variable (can be set more than once)                                                                                                            |
 | _--wait_                                    |   (bool)   | Wait until the pipeline finishes (defaults to false)                                                                                                           |
+| _--no-bash_                                 |   (bool)   | Execute commands without wrapping with bash (default to false)                                                                                                 |
```

Comment:
Is "wrapping" the right term  for this?

Reply:
I think we can be more explicit by saying "it will execute the command using the container's default shell instead if bash" ?